src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,17 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        List<Integer> list = new ArrayList<Integer>();
+        String[] stringArray = text.replaceAll("[^-0-9]", " ").trim().split(" ");
```

Comment:
```suggestion
        String[] stringArray = text.replaceAll("[^-0-9]+", " ").trim().split(" ");
```
this way you don't need check in loop below